shared/src/bundles/resumes/types/resume-share-response-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+type ResumeShareCreateResponseDto = {
+    id: string;
+};
+
+export { type ResumeShareCreateResponseDto };
```

Comment:
for me personally it's a bit confusing to see so many types with the same body inside, if the same type is using for another DTO class you can just do like so:

ResumeShareCreateResponseDto = ResumeShareDetailsGetRequestDto

or create a `base type` and then assign it as DTO, do not forget about Pick utility also

Reply:
Done